### ISSUE TITLE
Dedupe visits when sending to client

### ISSUE THREAD

**Issue by pdehaan** [mozilla/chronicle#258]
Currently, my Chronicle page looks like this (see **Figure 1** below), where I have multiple repeated entries for things like:
- Chronicle
- New Tab
- Problem loading page

... Or really any other page that you've hit refresh on, or happened to visit multiple times.

I know some of those will be solved by creative domain blocking on the client/server, but we may want to do some result aggregation for dupes (and combine dwell times, etc) on the API server before sending results back to the client.

![chronicle](https://cloud.githubusercontent.com/assets/557895/6029544/28355fd8-aba6-11e4-9a9d-2417de1d5e5e.png)
**Figure 1:** Chronicle

**Comment by 6a68** [comment]
The problems I see here are: (1) we should ignore schemes like 'about' and 'file' and 'chrome', (2) we should ignore the chronicle page itself. There's a pull request for the addon that addresses these, would be nice for the server to handle this as well, so I'll leave this as a server bug. Thanks @pdehaan!

**Comment by johngruen** [comment]
trippy

**Comment by pdehaan** [comment]
Well, not entirely related to "about"-style links. I can reproduce in latest "dev" server env with a random domain like cnn.com:

![mozilla_firefox](https://cloud.githubusercontent.com/assets/557895/6046651/fa1b400a-ac58-11e4-8794-399c98b78529.png)
**Figure 2:** The one where @pdehaan presses Refresh on cnn.com like it's going out of style.

I imagine this may cause the server go to into full panic mode when we start throwing gmail links into the mix, but we may need a bit more work on the plugin(s) to detect SPA pushstate behaviors before that is an issue.

**Comment by 6a68** [comment]
@pdehaan that's true, and it'll matter someday soon, though we need to um, pave the road before we put up guardrails. As a reminder, we just today are styling the dumbest possible feed, we're going to be playing with this for a while before we get to intelligently grouping related visits, either on server or client.
